plugin.rb

```diff
@@ -135,13 +151,32 @@ def check_post(bouncer, post)
     check_post(bouncer, post) if bouncer.should_check?(post)
   end
 
+  on(:post_voting_comment_created) do |comment, params|
+    bouncer = DiscourseAkismet::PostVotingCommentsBouncer.new
+    if bouncer.should_check?(comment)
+      # Store extra data for akismet
+      bouncer.store_additional_information(comment, params)
+      check_post(bouncer, comment)
+    end
+  end
+
+  on(:post_voting_comment_edited) do |comment, _, revisor|
+    next unless revisor.reviewable_content_changed?
+
+    editor = comment.last_editor
+    next if editor.is_system_user? || !editor.regular?
+
+    bouncer = DiscourseAkismet::PostVotingCommentsBouncer.new
+    check_post(bouncer, comment) if bouncer.should_check?(comment)
+  end
+
   # If a user is anonymized, support anonymizing their IPs
   on(:user_anonymized) do |args|
     user = args[:user]
     opts = args[:opts]
 
     if user && opts && opts.has_key?(:anonymize_ip)
-      sql = <<~SQL
+      sql1 = <<~SQL
```

Comment:
Could you name `sql1` and `sql2` to something meaningful instead? It's _kinda_ fine when it's just `sql` but when it gets indexed with numbers, it feels like a name should be there instead.